src/libs/actions/App.js

```diff
@@ -259,7 +262,8 @@ Onyx.connect({
                 previousUpdateIDFromServer,
                 lastUpdateIDAppliedToClient,
             });
-            getMissingOnyxUpdates(lastUpdateIDAppliedToClient, lastUpdateIDFromServer);
+            SequentialQueue.pause();
+            getMissingOnyxUpdates(lastUpdateIDAppliedToClient, lastUpdateIDFromServer).then(SequentialQueue.unpause);
```

Comment:
What will happen here if GetMissingOnyxUpdates returns a 423?

Reply:
Nevermind, I don't think 423 will ever reach the app since we're doing a reconnect app if it returns 423 in the backend.

Reply:
I think if there was a failure like that, then all of the requests in the queue would also fail with that error... so it doesn't seem helpful to unpause the queue. However, with the queue paused, there isn't really anyway for the user to recover from the error other than refreshing or killing the app. What about changing this to a `finally(unpause)` to at least ensure the queue won't get stuck.

Reply:
> What about changing this to a `finally(unpause)` to at least ensure the queue won't get stuck.

That feels safer to me! I have seen the App getting stuck permanently by a similar state that didn't allow further navigation. Using `finally` should help with avoiding that.

Reply:
I've gone ahead and done that so this has been UPDATED